### PR TITLE
Feature: rtl support

### PR DESCRIPTION
**Pull request by wuweiweiwu**
closes: #209 
closes: #325 
closes: #253 
closes: #311 

follow up to #331 

- [ ] documentation
- [ ] add storybook example

### CODE REVIEW

**Comment by lionben89** [comment]
Hi,

Can you please check our changes and pull it into master?

Thanks

File: src/__snapshots__/react-sortable-tree.test.js.snap [hunk 1]
```diff
@@ -60,7 +61,7 @@ exports[`<SortableTree /> should render tree correctly 1`] = `
           }
         >
           <div
-            className="rst__lineBlock rst__lineHalfHorizontalRight"
+            className="rst__lineBlock rst__lineHalfHorizontalRight "
```
Comment (wuweiweiwu) [inline src/__snapshots__/react-sortable-tree.test.js.snap]: Can you get rid of the extra space at the end?

File: src/tree-node.js [hunk 2]
```diff
@@ -81,7 +84,7 @@ class TreeNode extends Component {
         <div
           key={`pre_${1 + i}`}
           style={{ width: scaffoldBlockPxWidth }}
-          className={`${'rst__lineBlock'} ${lineClass}`}
+          className={`${'rst__lineBlock'} ${lineClass} ${directionClass}`}
```
Comment (wuweiweiwu) [inline src/tree-node.js]: you can use classnames here

File: src/__snapshots__/react-sortable-tree.test.js.snap [hunk 3]
```diff
@@ -49,6 +49,7 @@ exports[`<SortableTree /> should render tree correctly 1`] = `
       >
         <div
           className="rst__node"
+          direction={undefined}
```
Comment (wuweiweiwu) [inline src/__snapshots__/react-sortable-tree.test.js.snap]: why is this undefined in a lot of places?

**Comment by wuweiweiwu** [comment]
Wow @lionben89 I'm really impressed.

screen shot 2018-06-06 at 1 05 14 am

All that is left is adding documentation and a storybook example. If you don't want to add the storybook example I can do it.

**Pull request closed without merging**